modules/prometheus-exporter_oracle/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,14 @@
+module: oracle
+name: heartbeat
+
+transformation: false
+aggregation: ".mean(by=['hostname'])"
+
+exclude_not_running_vm: true
+
+signals:
+  signal:
+    metric: oracledb_Process_limits_value
```

Comment:
I did some tests with "oracledb_up" and it was not working than expected : I receive alerts of heartbeat whereas all is ok on db side (up). Since I'm using process checks and it seems that it's working correctly.

Reply:
I am pretty sure the problems are due to your tests, you probably dropped some dimensions so it leads to alerts, this is the same for any metric but perhaps you stopped to change your dimensions after the switch to this one

Reply:
yes, perhaps a simple workshop together may help me as I'm not totally aware with spunk usage / hearbeat etc. I will ping Jeremy if he could help me on it.

Reply:
yes feel free to join me on huddle. what you need to understand is: as soon as you change dimensions on your agent configuration you can change generated MTS too.

especially, if you remove or rename a dimension (key or value) the "old" mts will stop to send datapoint and a new mts will be created.

the heartbeat will so trigger an alert for the disappearing mts and starts to monitor the new one.
forcing aggregation will mitigate this risk because if you aggregate on `host` for example, no matter your changes on others dimensions, while this dimensions does not change you will never get alert.
that said, in general we recommend to not use aggregation from module side except if really required and give the user a variable to define it himself depending on his environment.